db/schema.rb

```diff
@@ -28,9 +28,9 @@
 
   create_table "trips", force: :cascade do |t|
     t.integer "duration"
-    t.integer "start_date"
+    t.string  "start_date"
     t.integer "start_station_id"
-    t.integer "end_date"
+    t.string  "end_date"
```

Comment:
I think this is fine as a string for now until we work out our strategy for dates, and then we can change it to a date data type.